proc_Hahn_echoph.py

```diff
@@ -10,38 +10,14 @@
 for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
         ('200302_alex_probe_water','test_equip','signal','spincore_Hahn_echoph_v1','microwaves off',(-5e3,5e3)),
         ]:
-    filename = date+'_'+id_string+'.h5'
-    nodename = 'signal'
-    s = nddata_hdf5(filename+'/'+nodename,
-            directory = getDATADIR(
-                exp_type = 'test_equip'))
-    nPoints = s.get_prop('acq_params')['nPoints']
-    nEchoes = s.get_prop('acq_params')['nEchoes']
-    #nPhaseSteps = s.get_prop('acq_params')['nPhaseSteps']
-    SW_kHz = s.get_prop('acq_params')['SW_kHz']
-    nScans = s.get_prop('acq_params')['nScans']
-    print(ndshape(s))
-    s.reorder('t',first=True)
-    t2_axis = s.getaxis('t')[0:2048]
-    s.setaxis('t',None)
-    s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
-    s.setaxis('ph2',r_[0.,2.]/4)
-    s.setaxis('ph1',r_[0.,1.,2.,3.]/4)
-    s.setaxis('t2',t2_axis)
-    s.setaxis('nScans',r_[0:nScans])
-    s.reorder('t2',first=False)
-    s.ft('t2',shift=True)
-    fl.next('raw data, chunked')
-    fl.image(abs(s))
-    s.ft(['ph1','ph2'])
-    fl.next('coherence')
-    fl.image(abs(s))
-    s = s['ph1',1]['ph2',0].C
-    s.mean('nScans')#,return_error=False)
-    fl.next('signal')
-    fl.plot(abs(s),label=label_str)
-    slice_f = (-5e3,5e3)
-    s = s['t2':slice_f].C
```

Comment:
here, you seemed to be stepping backwards -- specifically moving things from postproc back into the code??

Reply:
I was trying to resolve an older conflict on slack using an older processing code to keep apples to apples as you phrase it lol but forgot to convert back to the newest code. It is updated and runs well now